docs/Using Price Feeds/historical-price-data.html

```diff
@@ -31,7 +31,51 @@
 
 # Historical Rounds
 
+To get historical data, one needs to call the `getRoundData` [function](/docs/price-feeds-api-reference/#getrounddata) providing `roundId` as parameter.
 Data Feeds are updated in rounds. Rounds are identified by their `roundId`, which increases with each new round. The increase might not be monotonic. Knowing the `roundId` of a previous round allows contracts to consume historical price data.
+Technically, the `roundId` is computed as the following in the [proxy contract](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.7/dev/AggregatorProxy.sol):
+
+```solidity
+roundId = uint80((uint256(phaseId) << 64) | originalId);
+```
+
+where:
```

Comment:
```suggestion
where:
```
unclear what this is referring to

Reply:
will provide a detailed explanation with a sequence diagram